3574/2/helpers/session_state.rb

```diff
@@ -0,0 +1,27 @@
+module SessionStatus
+  private
+
+  def start_not_registered
```

Comment:
1. bad naming

Reply:
2. method should be responsible for 1 thing: either for responding or for defining wether user is registered